base/sparse/symmetric.jl

```diff
@@ -0,0 +1,72 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+function Symmetric(A::SparseMatrixCSC, uplo::Symbol=:U)
```

Comment:
This is not needed since it is equivalent to the default constructor: https://github.com/JuliaLang/julia/blob/063e8f110bb7f0ee61bbc166ad6d0af7e2113d87/base/linalg/symmetric.jl#L43